### ISSUE TITLE
Question: Vagrantfile is broken

### ISSUE THREAD

**Issue by madhur** [vitessio/vitess#9753]
### Question

```markdown
Is anyone maintaining the Vagrantfile for Vitess? I got an error while logging that GO version is 1.12 while Vitess requires Vitess 1.17.

Even after fixing that issue, there are numerous errors in whole provisioning process.
```

**Comment by frouioui** [comment]
I do not think we are actively maintaining the Vagranfile. At least it has been a long while since its last update.

@deepthi, is that something we want to start maintaining again?

**Comment by madhur** [comment]
I connected with @deepthi  over slack and she mentioned that Vagrantfile is not being maintained.

**Comment by deepthi** [comment]
Fixed in #9768
